server/webapp/WEB-INF/urlrewrite.xml

```diff
@@ -136,12 +136,19 @@
 
     <rule>
         <name>Rails API</name>
-        <from>^/(api/(?!admin/config).*)$</from>
+        <from>^/(api/((?!admin/config)(?!users/)).*)$</from>
```

Comment:
Ok, I can move it into a separate rule.

I was unable to run the tests even for a single rule, it takes a long time and always times out.  Anyways will add a test, will have to check if it breaks on CI.